archive/2025-10-23-dane-stuckey-openai-ciso-on-prompt-injection-risks-for-chatgpt-atlas.md

```diff
@@ -0,0 +1,158 @@
+
+We saw examples of browser agents from other vendors leaking private data in this way [identified by the Brave security team just yesterday](https://simonwillison.net/2025/Oct/21/unseeable-prompt-injections/).
+
+> Our long-term goal is that you should be able to trust ChatGPT agent to use your browser, **the same way you’d trust your most competent, trustworthy, and security-aware colleague** or friend.
+
+This is an interesting way to frame the eventual goal, describing an extraordinary level of trust and competence.
+
+As always, a big difference between AI systems and a human is that an AI system [cannot be held accountable for its actions](https://simonwillison.net/2025/Feb/3/a-computer-can-never-be-held-accountable/). I’ll let my trusted friend use my logged-in browser only because there are social consequences if they abuse that trust!
+
+> We’re working hard to achieve that. For this launch, we’ve performed extensive red-teaming, implemented novel model training techniques to reward the model for ignoring malicious instructions, **implemented overlapping guardrails and safety measures**, and added new systems to detect and block such attacks. However, **prompt injection remains a frontier, unsolved security problem, and our adversaries will spend significant time and resources to find ways to make ChatGPT agent fall for these attacks**.
+
+I’m glad to see OpenAI’s CISO openly acknowledging that prompt injection remains an unsolved security problem (three years after we [started talking about it](https://simonwillison.net/2022/Sep/12/prompt-injection/)!).
+
+That “adversaries will spend significant time and resources” thing is the root of why I don’t see guardrails and safety measures as providing a credible solution to this problem.
+
+As I’ve written before, in application security [99% is a failing grade](https://simonwillison.net/2023/May/2/prompt-injection-explained/#prompt-injection.015). If there’s a way to get past the guardrails, no matter how obscure, a motivated adversarial attacker is going to figure that out.
+
+Dane goes on to describe some of those measures:
+
+> To protect our users, and to help improve our models against these attacks:
+>
+> 1. We’ve prioritized rapid response systems to help us quickly identify block attack campaigns as we become aware of them.
+
+I like this a lot. OpenAI have an advantage here of being a centralized system—they can monitor their entire user base for signs of new attack patterns.
+
+It’s still bad news for users that get caught out by a zero-day prompt injection, but it does at least mean that successful new attack patterns should have a small window of opportunity.
```

Comment:
I don't like this term zero-day prompt. Folks, we now auto-generate those prompts. We always have infinite prompts that work. Stop trying to build a perimeter it isn't working.